data/Te-Stack.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "Quincy Oghenetejiri Ukumakube",
+  "bio": "Software Developer and Technical Writer"
```

```suggestion
  "bio": "Software Developer and Technical Writer",
```
Please add a comma here